packages/financial-templates-lib/src/proxy-transaction-handler/DSProxyManager.js

```diff
@@ -10,7 +10,16 @@ class DSProxyManager {
    * @param {String} account Ethereum account from which to send txns.
    * @param {String} dsProxyFactoryAddress address of the DSProxy factory to create new DSProxies and the like.
    */
-  constructor({ logger, web3, gasEstimator, account, dsProxyFactoryAddress, dsProxyFactoryAbi, dsProxyAbi }) {
+  constructor({
+    logger,
+    web3,
+    gasEstimator,
+    account,
+    dsProxyFactoryAddress,
+    dsProxyFactoryAbi,
+    dsProxyAbi,
+    availableAccounts = 1
```

Comment:
only in the case of the `DSProxyManager` do we need `availableAccounts` as this only works with DSProxy enabled bots. If not set, defaults to 1 (first and only EOA).